lib/panda/client.rb

```diff
@@ -78,9 +85,9 @@ def get_token(path, params = {})
       Panda::Item.new(Panda.make_request(request))
     end
 
-    def get_collection(path, params = {})
+    def get_collection(path, collection_key, params = {})
```

Comment:
Can we move this new param to the last position and make it optional with default value 'list'?